src/components/InventoryTable/EntityTableToolbar.js

```diff
@@ -342,7 +327,7 @@ const EntityTableToolbar = ({
                     }))
                 }
             }}
-            { ...isFilterSelected() && hasAccess && { activeFiltersConfig: constructFilters() } }
```

Comment:
What was the reasoning for removing this? ...I'm very suspicious of this change. :D

Reply:
My thinking is that now we're setting activeFiltersConfig regardless of if a filter is selected or not. activeFiltersConfig is only applied if a filter is selected. We want the 'reset filter' button to always display regardless of if a filter is selected. So now we only need to check if the user has access.

Reply:
My thinking is that now we're setting activeFiltersConfig regardless of if a filter is selected or not. activeFiltersConfig is only applied if a filter is selected. We want the 'reset filter' button to always display regardless of if a filter is selected. So now we only need to check if the user has access.